src/dice/ffi/metall.h

```diff
@@ -5,11 +5,13 @@
 #include <stddef.h>
 
 #include <metall/logger_interface.h>
+#include <metall/defs.hpp>
```

Comment:
This is included so that METALL_DEFAULT_CAPACITY is available, right?

Reply:
correct, I checked in the header, there is only C stuff in there (mostly `#define`s). No C++.